src/Services/InvoiceService.php

```diff
@@ -40,7 +40,7 @@ public function paginator($params = [])
     /**
      * @param array|ArrayObject $params
      *
-     * @return Invoice[]|Collection
+     * @return Invoice[]
```

Comment:
Other services have `Collection`, are you sure it's a valid change?
```suggestion
     * @return Invoice[]|Collection
```

Reply:
I posted in another repo, please check it there.

Reply:
It actually returns a Collection instance which contains Invoice enttites:

https://github.com/Rebilly/rebilly-php/blob/4b1b3cefb56c1a61146ea16c808ffa8ed9000422/src/Entities/Schema.php#L94

Reply:
Reverted, thanks